astro_2.0/components/BreadCrumbs/BreadCrumbs.tsx

```diff
@@ -84,7 +84,7 @@ export const BreadCrumbs: React.VFC<BreadCrumbsProps> = ({
       </li>,
       !isLastItem && (
         <li className={styles.item}>
-          <Icon name={separator} width={16} />
+          <Icon name={separator} width={16} style={{ display: 'inline' }} />
```

Comment:
Chakra overrode `svg` to always use `display: "block"`, need to keep this `inline` for proper styling.